system/Commands/Utilities/FilterCheck.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Utilities;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use CodeIgniter\Commands\Utilities\Routes\FilterCollector;
+use Config\Services;
+
+/**
+ * Check filters for a route.
+ */
+class FilterCheck extends BaseCommand
+{
+    /**
+     * The group the command is lumped under
+     * when listing commands.
+     *
+     * @var string
+     */
+    protected $group = 'CodeIgniter';
+
+    /**
+     * The Command's name
+     *
+     * @var string
+     */
+    protected $name = 'filter:check';
+
+    /**
+     * the Command's short description
+     *
+     * @var string
+     */
+    protected $description = 'Check filters for a route.';
+
+    /**
+     * the Command's usage
+     *
+     * @var string
+     */
+    protected $usage = 'filter:check <HTTP method> <route>';
+
+    /**
+     * the Command's Arguments
+     *
+     * @var array<string, string>
+     */
+    protected $arguments = [
+        'method' => 'The HTTP method. get, post, put, etc.',
+        'route'  => 'The route (URI path) to check filtes.',
+    ];
+
+    /**
+     * the Command's Options
+     *
+     * @var array
+     */
+    protected $options = [];
+
+    /**
+     * @return int exit code
+     */
+    public function run(array $params)
+    {
+        if (count($params) !== 2) {
```

Comment:
I think it would be better to use `! isset($params[1])` to check if all arguments are present. If, for example, a user supplied only the first argument then, out of the blue, supplies an option (e.g. `--force`) then would `count($params)` be 2?

Reply:
Good suggestion!
Fixed.